tests/failpoints/cases/test_unsafe_recovery.rs

```diff
@@ -209,23 +209,16 @@ fn test_unsafe_recover_wait_for_snapshot_apply() {
     let store2_peer = find_peer(&region, nodes[1]).unwrap().to_owned();
     cluster.must_transfer_leader(region.get_id(), store2_peer);
     cluster.stop_node(nodes[1]);
-    let (raft_gc_triggered_tx, raft_gc_triggered_rx) = mpsc::bounded::<()>(1);
```

Comment:
Why remove this?

Reply:
I think it's unnecessary (@v01dstar can help scrutinize) and add overhead because `worker_gc_raft_log` needs to wait. On IDC this test fails frequently due to `raft_gc_finished_rx` timeout.

Reply:
@Connor1996 PTAL

Reply:
I increased the timeout here https://github.com/tikv/tikv/pull/12810, should be able to reduce the flakiness.